src/QueryBuilder/QueryBuilder/QueryBuilderModal/RepeatableFields/RepeatableFields.js

```diff
@@ -96,6 +96,8 @@ export const RepeatableFields = ({ source, setSource }) => {
         onAdd={() => {}}
         hasMargin={false}
         renderField={(row, index) => {
+          console.log(row.value);
```

Comment:
Was this console log left in for debugging?

Reply:
Thanks @ryandberger for noting that. I remove it already